taccsite_cms/settings.py

```diff
@@ -1,104 +1,172 @@
-# taccsite_cms/settings.py
 
 """
-Django settings for taccsite_cms project.
-
+Django settings
 Generated by 'django-admin startproject' using Django 1.11.22.
-
 For more information on this file, see
 https://docs.djangoproject.com/en/2.2/topics/settings/
-
 For the full list of settings and their values, see
 https://docs.djangoproject.com/en/2.2/ref/settings/
 """
 
 import logging
 import os
 from glob import glob
+import ldap
+from django_auth_ldap.config import LDAPSearch, GroupOfNamesType
 
-
+SECRET_KEY = 'CHANGE_ME'
 def gettext(s): return s
 
+DATA_DIR = os.path.dirname(os.path.dirname(__file__))
 
-# Import secret values dynamically without breaking portal.
```

Comment:
This seems like a major refactor. Could we pull out what is required by this PR from the rest of the refactor?

Reply:
This is not a major refactor (one file changed in the core project, same for custom sites), but it is a change that puts us in a position to easily remove optional features from the core cms and push those out to the custom sites that need them (Something we've already discussed as good and can be done with minimal change after this change).  

Again, this is very low risk change that will go through testing after review. Any changes in settings not accounted for here will be easy to identify and update in testing.